src/reactapp/src/context/Cart/cartDispatcher.js

```diff
@@ -27,6 +27,7 @@ const dispatchMapper = {
   placeOrder: placeOrderAction,
   mergeCarts: mergeCartsAction,
   setOrderInfo: setOrderInfoAction,
+  setCartInfo: setCartInfoAction,
```

Comment:
You need to import `setCartInfoAction` in this file too

Reply:
Actually I will create a seperate PR for this change, I prematurely committed the change on the wrong branch.. Thanks and stay tuned.